pkg/gitproviders/dryrun.go

```diff
@@ -0,0 +1,83 @@
+package gitproviders
+
+import "github.com/fluxcd/go-git-providers/gitprovider"
+
+type dryrunProvider struct {
+	provider GitProvider
+}
+
+func NewDryRun(_ Config) (GitProvider, error) {
```

Comment:
Why do we need a `Config` object as an argument here?

Reply:
we don't, removing it.